app/quest-boost/claim/[boostId]/page.tsx

```diff
@@ -136,7 +146,13 @@ export default function Page({ params }: BoostQuestPageProps) {
                     )}
                   </div>
                   <div className={styles.claim_button_text}>
-                    <p className={styles.claim_amount}>{boost?.amount}</p>
+                    <p className={styles.claim_amount}>
+                      {boost
+                        ? parseInt(
```

Comment:
put that in a const for it to be cleaner

Reply:
done